dashboard/test/models/pd/workshop_test.rb

```diff
@@ -1202,6 +1202,34 @@ class Pd::WorkshopTest < ActiveSupport::TestCase
     assert_equal csp_workshops[0], Pd::Workshop.where(course: COURSE_CSP).nearest_attended_or_enrolled_in_by(teacher)
   end
 
+  test 'pilot csf201 workshop does not get exit email' do
+    skip "Skip this test from #{CSF_201_PILOT_END_DATE}" unless
```

Comment:
This functionality should still work after this date - we never want to send those emails for these workshops, right?

Reply:
At certain point we will know that there is no pilot workshop (the ones that start before 5/20) that hasn't ended, then this test loses its value and we don't need to spend resources on running it. I can set that point to `CSF_201_PILOT_END_DATE + 7.days` if you agree.

Reply:
Ohh I misread this - I thought it was about surveys, not emails. Let's add a follow up jira item to remove these tests and the code related to the csf 201 pilot workshops after this date.

Reply:
will do

Reply:
Add https://codedotorg.atlassian.net/browse/PLC-215